<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibFrame</class>
 <widget class="QFrame" name="CalibFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt;
QLineEdit {
	min-Height: 32px;
}
QPushButton { 
	min-Height:50px;
	min-Width:10px;
}
QLabel {
	 min-Height:32px;
}
</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="dsbConc4">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="leAbs2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="dsbConc3">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QSpinBox" name="sbSample2">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QComboBox" name="cbSelect1">
     <item>
      <property name="text">
       <string>不标定</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>待标定</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>已标定</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lbCali">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>标样</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lbConc">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>浓度(mg/L)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lbAbs">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>吸光度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="lbRange">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>量程切换</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="lbPipe">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>取液管道</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="lbSelection">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>选择</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbCali0">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>零标样</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="leAbs0">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="cbRange0">
     <item>
      <property name="text">
       <string>0.0-2.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.0-10.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>自定义量程</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QComboBox" name="cbPipe0"/>
   </item>
   <item row="1" column="7">
    <widget class="QComboBox" name="cbSelect0">
     <item>
      <property name="text">
       <string>不标定</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>待标定</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>已标定</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbCali1">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>标样1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="leAbs1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="cbRange1">
     <item>
      <property name="text">
       <string>0.0-2.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.0-10.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>自定义量程</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="7">
    <widget class="QComboBox" name="cbSelect5">
     <item>
      <property name="text">
       <string>不标定</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>待标定</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>已标定</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lbCali2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>标样2</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QComboBox" name="cbRange2">
     <item>
      <property name="text">
       <string>0.0-2.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.0-10.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>自定义量程</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QComboBox" name="cbSelect2">
     <item>
      <property name="text">
       <string>不标定</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>待标定</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>已标定</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lbCali3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>标样3</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="leAbs3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QComboBox" name="cbRange3">
     <item>
      <property name="text">
       <string>0.0-2.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.0-10.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>自定义量程</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QSpinBox" name="sbWater1">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lbCali4">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>标样4</string>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QComboBox" name="cbSelect3">
     <item>
      <property name="text">
       <string>不标定</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>待标定</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>已标定</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="leAbs4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QComboBox" name="cbRange4">
     <item>
      <property name="text">
       <string>0.0-2.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.0-10.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>自定义量程</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="7">
    <widget class="QComboBox" name="cbSelect4">
     <item>
      <property name="text">
       <string>不标定</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>待标定</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>已标定</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lbCali5">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>标样5</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="8">
    <widget class="Line" name="line2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="leAbs5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QComboBox" name="cbRange5">
     <item>
      <property name="text">
       <string>0.0-2.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.0-10.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>自定义量程</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="0" colspan="8">
    <widget class="Line" name="line1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="8">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QPushButton" name="pbTrain">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>训练</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pbDo">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>执行</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pbSave">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pbReset">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0" colspan="8">
    <widget class="QLabel" name="lbResult">
     <property name="text">
      <string>拟合结果：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QComboBox" name="cbPipe1"/>
   </item>
   <item row="5" column="4">
    <widget class="QComboBox" name="cbPipe4"/>
   </item>
   <item row="3" column="4">
    <widget class="QComboBox" name="cbPipe2"/>
   </item>
   <item row="4" column="4">
    <widget class="QComboBox" name="cbPipe3"/>
   </item>
   <item row="6" column="4">
    <widget class="QComboBox" name="cbPipe5"/>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="dsbConc1">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="dsbConc2">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QSpinBox" name="sbSample4">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QSpinBox" name="sbSample1">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QSpinBox" name="sbSample3">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="dsbConc5">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="5" colspan="2">
    <widget class="QLabel" name="lbDilu">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>标液:蒸馏水</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QSpinBox" name="sbSample5">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="dsbConc0">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QSpinBox" name="sbWater0">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QSpinBox" name="sbWater2">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QSpinBox" name="sbWater3">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="5" column="6">
    <widget class="QSpinBox" name="sbWater4">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="6" column="6">
    <widget class="QSpinBox" name="sbWater5">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QSpinBox" name="sbSample0">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
